Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -187,41 +189,13 @@ skipButtonConfig state =
         , margin = MarginTop 22
         , id = "SkipButton"
         , enableLoader = (JB.getBtnLoader "SkipButton")
+        , visibility = if not state.data.ratingViewState.issueFacedView || state.data.ratingViewState.doneButtonVisibility then VISIBLE else GONE
```

Comment:
as `state.data.ratingViewState.issueFacedView` this is being used thrice in this function, better to make a local scoped variable and use it for better code readability